### PR TITLE
Fixes #6363 verify telehealth settings button

### LINKED ISSUE #6363: Telehealth Verify Settings button
Add a Telehealth Verify Settings button that checks both the provisioning and the video communication bridge settings.  

Right now there isn't a way for someone to verify the telehealth is setting up properly without creating appointments, users, and going through a full telehealth session.  We need more information that helps users to know if the telehealth is configured properly and an option to check that the settings are properly configured.  This ideally should be on the same screen as the telehealth settings configuration.

### PR DESCRIPTION
**Pull request by adunsulag**
Added the ability to verify the telehealth configuration settings to the Telehealth settings.

Also added a new GlobalSetting option to provide html sections to the global settings.  That way information boxes and any other custom large display of code for the settings can be displayed in the GlobalsConfiguration area.

### CODE REVIEW

**Comment by stephenwaite** [comment]
excited_elephant

File: interface/modules/custom_modules/oe-module-comlink-telehealth/src/Util/AuthUtils.php [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace Comlink\OpenEMR\Modules\TeleHealthModule\Util;
+
+class AuthUtils
```
Comment (bradymiller) [inline interface/modules/custom_modules/oe-module-comlink-telehealth/src/Util/AuthUtils.php:5]: note there is also a AuthUtils class in core OpenEMR, which could cause confusion

File: interface/modules/custom_modules/oe-module-comlink-telehealth/templates/comlink/admin/telehealth_footer_box.html.twig [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+</script>
+<div class="alert alert-info">
+    <details>
+        <summary class="h4">
+            {{ "Telehealth Configuration Settings Check"|xlt }}
+        </summary>
+        <ul>
+            <li class="h5">
+                {{ "Config"|xlt }} -> {{ "Telehealth"|xlt }}
+                <input type="button" class="btn btn-primary btn-verify-comlink-configuration" value="{{ "Check Telehealth Configuration" }}" />
```
Comment (bradymiller) [inline interface/modules/custom_modules/oe-module-comlink-telehealth/templates/comlink/admin/telehealth_footer_box.html.twig:87]: translate

File: interface/modules/custom_modules/oe-module-comlink-telehealth/templates/comlink/admin/telehealth_footer_box.html.twig [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+                <p class="d-none section-verify-configuration-settings"></p>
+                <p>
+                    {{ "The following setting must be configured in the Config Globals section for Telehealth to be enabled" }}
```
Comment (bradymiller) [inline interface/modules/custom_modules/oe-module-comlink-telehealth/templates/comlink/admin/telehealth_footer_box.html.twig:90]: translate

File: interface/modules/custom_modules/oe-module-comlink-telehealth/templates/comlink/admin/telehealth_footer_box.html.twig [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+            </li>
+            <li class="h5">
+                {{ "Config"|xlt }} -> {{ "Portal"|xlt }} -
+                {% if isThirdPartyConfigurationSetup %}
+                    <span class="text-success">{{ "Configured"|xlt }}</span>
+                {% else %}
+                    <span class="text-danger">{{ "Incorrect"|xlt }}</span>
+                {% endif %}
+                <p>
+                    {{ "The following settings must be configured in the Config Portal section for patients to join the telehealth session" }}
```
Comment (bradymiller) [inline interface/modules/custom_modules/oe-module-comlink-telehealth/templates/comlink/admin/telehealth_footer_box.html.twig:120]: translate

File: interface/modules/custom_modules/oe-module-comlink-telehealth/templates/comlink/admin/telehealth_footer_box.html.twig [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+            </li>
+            <li class="h5">
+                {{ "Config"|xlt }} -> {{ "Notifications"|xlt }} -
+                {% if emailNotificationsConfigured %}
+                    <span class="text-success">{{ "Configured"|xlt }}</span>
+                {% else %}
+                    <span class="text-danger">{{ "Incorrect"|xlt }}</span>
+                {% endif %}
+                <p>
+                    {{ "The following settings must be configured in the Config Notifications section for Telehealth invitations to go out" }}
```
Comment (bradymiller) [inline interface/modules/custom_modules/oe-module-comlink-telehealth/templates/comlink/admin/telehealth_footer_box.html.twig:135]: translate

File: interface/super/templates/field_html_display_section.php [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+            $fldarr
+        );
+        if (!empty($displaySection)) {
+            echo $displaySection;
+        }
+    } catch (\Exception $e) {
+        ob_end_clean();
+        (new SystemLogger())->errorLogCaller($e->getMessage(), ['trace' => $e->getMessage()]);
+        echo xlt("Error in rendering html display section.")
+            . xlt("Field name") . " '" . $fldname . "'";
```
Comment (bradymiller) [inline interface/super/templates/field_html_display_section.php:26]: text $fldname (unless there are html elements in this variable)

File: interface/modules/custom_modules/oe-module-comlink-telehealth/src/Util/AuthUtils.php (hunk 1 above)
Comment (adunsulag) [reply interface/modules/custom_modules/oe-module-comlink-telehealth/src/Util/AuthUtils.php:5]: It is namespaced, but I went ahead and renamed it to TelehealthAuthUtils to mitigate confusion.

File: interface/modules/custom_modules/oe-module-comlink-telehealth/templates/comlink/admin/telehealth_footer_box.html.twig (hunk 2 above)
Comment (adunsulag) [reply interface/modules/custom_modules/oe-module-comlink-telehealth/templates/comlink/admin/telehealth_footer_box.html.twig]: Fixed

File: interface/modules/custom_modules/oe-module-comlink-telehealth/templates/comlink/admin/telehealth_footer_box.html.twig (hunk 4 above)
Comment (adunsulag) [reply interface/modules/custom_modules/oe-module-comlink-telehealth/templates/comlink/admin/telehealth_footer_box.html.twig:120]: Fixed.

File: interface/modules/custom_modules/oe-module-comlink-telehealth/templates/comlink/admin/telehealth_footer_box.html.twig (hunk 5 above)
Comment (adunsulag) [reply interface/modules/custom_modules/oe-module-comlink-telehealth/templates/comlink/admin/telehealth_footer_box.html.twig]: Fixed.

File: interface/super/templates/field_html_display_section.php (hunk 6 above)
Comment (adunsulag) [reply interface/super/templates/field_html_display_section.php]: Fixed.

**Comment by adunsulag** [comment]
I've fixed the recommended changes.  If everything passes the CI engine I'll bring this in as we need this for the onetime work we are doing.

**Comment by bradymiller** [comment]
2DV-2

**Pull request merged**